.circleci/config.yml

```diff
@@ -331,6 +331,7 @@ jobs:
             --parallel-mode \
             --module pytest \
             raiden/tests \
+            -s \
```

Comment:
I don't think we need this. This will clutter the output of the test progress.

Reply:
Agreed, I have some more tests which might timeout in the CI, I'll remove it once they run.